### DISCUSSION TITLE
Converting Markdown to DITA using Dita-ot

### DISCUSSION THREAD

**Discussion by Yazan0930** [dita-ot/dita-ot#4224] [Q&A]
I'm looking to convert Markdown to DITA using the DITA-OT tool. However, DITA contains a greater variety of inline elements compared to Markdown, which only supports a basic flow. I'm wondering if there's a way to incorporate my own Markdown syntax so that it can be properly generated into DITA using DITA-OT?

**Comment by raducoravu** [comment]
The bundled Markdown to DITA conversion functionality is described here:
https://www.dita-ot.org/dev/topics/markdown-input.html
I have not tested this but recently, the latest release of DITA Open Toolkit 4.1 and the Markdown conversion plugin allows you to set up your own Markdown parser and take control over how the Markdown gets converted to DITA XML with your own Java code:
https://github.com/jelovirt/org.lwdita/pull/141
